!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      hover_thrust_factor: 0.058
      id: 0
      name: Default
      p_i_acc_error_limit: 0.6
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      position_d_xy: 1.0
      position_d_z: 0.76
      position_i_xy: 0.16
      position_i_z: 0.68
      position_p_xy: 4.0
      position_p_z: 2.3
      state: true
      type: ''
      v_i_acc_error_limit: 0.5
      velocity_d_xy: 1.34
      velocity_d_z: 1.34
      velocity_i_xy: 0.16
      velocity_i_z: 0.38
      velocity_p_xy: 5.0
      velocity_p_z: 2.0
    state: []
  hover_thrust_factor: 0.058
  p_i_acc_error_limit: 0.6
  position_d_xy: 1.0
  position_d_z: 0.76
  position_i_xy: 0.16
  position_i_z: 0.68
  position_p_xy: 4.0
  position_p_z: 2.3
  v_i_acc_error_limit: 0.5
  velocity_d_xy: 1.34
  velocity_d_z: 1.34
  velocity_i_xy: 0.16
  velocity_i_z: 0.38
  velocity_p_xy: 5.0
  velocity_p_z: 2.0
state: []
